fineract-provider/src/main/java/org/apache/fineract/portfolio/search/service/SearchUtil.java

```diff
@@ -131,7 +135,6 @@ public static String validateToJdbcColumn(String column, Map<String, ResultsetCo
             errors.add(parameterErrorWithValue("error.msg.column.empty", "Column filter is empty", API_PARAM_COLUMN, null));
         }
         if (column != null) {
-            SQLInjectionValidator.validateDynamicQuery(column);
```

Comment:
We dont need this check anymore?

Reply:
no we do not, if the column was found in the headers list, then no need to validate it